yt/utilities/grid_data_format/conversion/conversion_athena.py

```diff
@@ -450,7 +451,7 @@ def write_to_gdf(self, fn, grid):
 
         ## --------- Store Grid Data --------- ##
 
-        g0 = data_g.create_group("grid_%010i" % 0)
+        g0 = data_g.create_group("grid_{0:010}")
```

Comment:
this one is a little suprising perharps. Effectively this is equivalent to hardcoding `'0000000000'` or, perharps more clearly `'0'*10`. What's disturbing is that the first `0` is sort of meaningless: one would get the same result from `f"{'':010}"`